chill-scala/src/test/scala/com/twitter/chill/RegistrationIdsSpec.scala

```diff
@@ -0,0 +1,179 @@
+package com.twitter.chill
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class RegistrationIdsSpec extends WordSpec with Matchers {
+  """
+    |Projects using chill to long term persist serialized data (for example in event
```

Comment:
so this test protects the future, but not people in the past right? They all get broken right?

Could we work around that? Maybe add a new registrar that people can opt in to adding?

Reply:
Right, this would break compatibility for projects that have already used chill for long term storage. Defining a separate registrar is OK for me (although the order of entries in the registrar will be less orderly that way).

I'm thinking along these lines currently:
* Rename AllScalaRegistrar -> AllScalaRegistrar_0_9_2 for those who need the old registrations.
* Create a new AllScalaRegistrar that first registers everything from AllScalaRegistrar_0_9_2, then registers all the new items from this PR.
Does this sound OK?